src/index.html

```diff
@@ -11,7 +11,142 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+       
+        <label class="form">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form">
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            autocomplete="off"
+          >
+        </label>
+
+
+        <label class="form">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form">
+          Full date of birth:
+          <input type="date" name="date">
+        </label>
+
+        <label class="form">
+          I accept the term of the agreement
+          <input type="checkbox" name="accept">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="form">
+          E-mail:
+          <input 
+            type ="email"
+            name ="email"
+            placeholder="email@example.com"
+          >
+        </label>
+    
+        <label class="form">
+          Password:
+          <input type ="password" name ="password">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+   
+        <label class="form">
+          Do you love cats?
+          <input 
+            type ="radio" 
+            name ="yesOrNo" 
+            value="yes"
+          >
+           Yes
+          <input 
+            type ="radio" 
+            name ="yesOrNo" 
+            value="no"
+          >
+           No
+        </label>
```

Comment:
this label doesn't work correctly, now I can choose only yes, let's add separate labels for each input
```suggestion
          Do you love cats?
          <label>
            <input 
              type ="radio" 
              name ="yesOrNo" 
              value="yes"
            >
             Yes
           </label>
           
           <label>
            <input 
              type ="radio" 
              name ="yesOrNo" 
              value="no"
            >
           No
        </label>
```